dist/index.d.ts

```diff
@@ -55,6 +55,16 @@ export declare class ChainOps {
      * @param filter Filter which subscriptions you get back
      */
     listSubs(filter: watcher.IListFilter): Promise<any[]>;
+    /**
+     * Makes a call to watcher logging the address to the bloom
+     * @param address string of address e.g. 0x123
+     */
+    addAddressToPendingBloom(address: string): Promise<any>;
```

Comment:
❔ Wondering if it's necessary/useful for SDK users to know that a bloom filter is involved.  If we expose the implementation detail here, then it becomes part of the public "interface" of the library — SDK users may write their code in such a way as to take advantage of some aspect of how bloom filters work, which would lead to unexpected breakage if we ever decide to use something other than bloom filters in the future.

Reply:
Hmm, I'm thinking of keeping it as they should know that they could get false-positives?
Would you rather that be explained in the doc block and remove it from the function name?

Reply:
Ah, good thinking!  I would expect that most developers – even blockchain-savvy ones – will probably be unfamiliar with how bloom filters work; being explicit about what to expect will probably lead to less confusion.

It would be cool to also document that it uses bloom filters under-the-hood, but I think we should also note that it could be subject to change in future updates.

Reply:
done :)